Tasks/CopyFilesToMachines/CopyFilesToMachines.ps1

```diff
@@ -20,30 +20,99 @@ Write-Verbose "cleanTargetBeforeCopy = $cleanTargetBeforeCopy" -Verbose
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 
-$connection = Get-VssConnection -TaskContext $distributedTaskContext
+	# Default + constants #
+$defaultWirmPort = '5985'
+$defaultSkipCACheckOption = '-SkipCACheck'
```

Comment:
Default option cannot be SkipCA. SkipCA should be set only if user specifies it.